onboarding-carts/jmeter/basiccheck.jmx

```diff
@@ -9,7 +9,7 @@
         <collectionProp name="Arguments.arguments">
           <elementProp name="SERVER_URL" elementType="Argument">
             <stringProp name="Argument.name">SERVER_URL</stringProp>
-            <stringProp name="Argument.value">127.0.0.1</stringProp>
+            <stringProp name="Argument.value">carts.sockshop-staging.public.demo.keptn.sh</stringProp>
```

Comment:
Did you intend to put this hostname in  (instead of 127.0.0.1)?

Reply:
yes - that was intended. makes it easier for us to test.